### DISCUSSION TITLE
BigQuery Timestamp formatting

### DISCUSSION THREAD

**Discussion by jeff-bradley** [dbeaver/dbeaver#35737] [Q&A] [answered]
### Description

I am unable to change any Bigquery formatting for the field with a timestamp data type. Ultimately I am having an issue with it always being set to my Local time which is incorrect. I have a table with multiple BQ TIMESTAMP fields and 1 field with a DATETIME datatype. The DATETIME field works as expected and I can set the timezone accordingly but anything with a native TIMESTAMP datatype always seems to get adjusted to my local time. I have tried all kinds of settings to get this value to show the proper UTC time which is what is stored in BQ. I do not have this issue in any of my other data sources like postgres or MSSQL, the same table in mssql show the correct value in DBeaver.

In the screenshot believe this is the Default because for all data formats, The only way I can see the true value is to cast as a string type.

![image](https://github.com/user-attachments/assets/5ae1e638-3ce6-4eaa-8782-c9744e998bab)
![image](https://github.com/user-attachments/assets/7ca6c2d0-a730-4672-a774-292c31caa430)

### DBeaver Version

Enterprise 24.2.0

### Operating System

Windows 11

### Database and driver

Database: BigQuery 
https://www.googleapis.com/bigquery/v2

### Steps to reproduce

anytime view the data for timestamp fields its always defaults to localtime

### Additional context

the Restore Default Button in Data Formats seems to do nothing.

**Comment by E1izabeth** [comment, marked as answer]
Hi, @jeff-bradley 
Could you try to specify the UTC timezone in Preferences —> User interface?
![image](https://github.com/user-attachments/assets/54f26e0c-d308-48bc-a8b0-e6b8c3b2d383)

**Comment by jeff-bradley** [comment]
@E1izabeth That seems to work how I want it to. Weird that BQ was the only one affected. Thank you!
